trustpinning/certs_test.go

```diff
@@ -700,28 +576,51 @@ func filestoreWithTwoCerts(t *testing.T, gun, keyAlg string) (
 
 		certificates[i] = cert
 	}
-	return tempBaseDir, certStore, cryptoService, certificates
+	return tempBaseDir, cryptoService, certificates
 }
 
 func testValidateSuccessfulRootRotation(t *testing.T, keyAlg, rootKeyType string) {
 	// The gun to test
 	gun := "docker.com/notary"
 
-	tempBaseDir, certStore, cs, certificates := filestoreWithTwoCerts(t, gun, keyAlg)
+	tempBaseDir, err := ioutil.TempDir("", "notary-test-")
 	defer os.RemoveAll(tempBaseDir)
-	origRootCert := certificates[0]
-	replRootCert := certificates[1]
+	require.NoError(t, err, "failed to create a temporary directory: %s", err)
 
-	// Add the old root cert part of trustedCertificates
-	certStore.AddCert(origRootCert)
+	fileKeyStore, err := trustmanager.NewKeyFileStore(tempBaseDir, passphraseRetriever)
```

Comment:
Totally non-blocking:  Just wondering if we could use a memory file store here?  Only reason would be minimally less IO during the tests.  Won't have much impact at all, though, since time taken for the client tests pretty much eclipse everything else.